build/local/README.md

```diff
@@ -1,3 +1,13 @@
+## Prerequisites 
+Prior to following the instructions here, make sure of the following steps to ensure a smooth process - 
+ - You should have `go` installed on your machine. You can find the installation instructions for your OS [here](https://go.dev/doc/install). 
+    - Verify the installation by running `go version`.
```

Comment:
don't think we need to include the steps to verify go was installed

Reply:
I can remove the verification steps